docs/reference/motlp.md

```diff
@@ -177,27 +177,30 @@ When creating a Kubernetes secret, always encode the full string in Base64, incl
 
 ::::::
 
-% ## Rate limiting and throttling
-% 
-% The following rate limits apply to the {{motlp}}:
-% 
-% * n requests per second per API key
-% * m requests per minute per API key
-% 
-% If you exceed these limits, you will receive an HTTP 429 status code.
-%
-% ## Data retention
-%
-% Telemetry is stored in OTel-specific data streams. The separation is implemented by adding an `.otel` suffix to the `data_stream.dataset` for data streams that contain OTel signal data. This allows to have a separate index template for OTel data, such as `logs-*.otel-*`.
-%
-% To configure the data retention for each data stream from the **Streams** screen, refer to [Managed data retention](docs-content://solutions/observability/logs/streams/management/retention.md).
-%
 ## Failure store
 
 The {{motlp}} endpoint is designed to be highly available and resilient. However, there are some scenarios where data might be lost or not sent completely. The [Failure store](docs-content://manage-data/data-store/data-streams/failure-store.md) is a mechanism that allows you to recover from these scenarios.
 
 The Failure store is always enabled for {{motlp}} data streams. This prevents ingest pipeline exceptions and conflicts with data stream mappings. Failed documents are stored in a separate index. You can view the failed documents from the **Data Set Quality** page. Refer to [Data set quality](docs-content://solutions/observability/data-set-quality-monitoring.md).
 
+## Routing logs to dedicated datasets
+
+You can route logs to dedicated datasets by setting the `data_stream.dataset` attribute to the log record. This attribute is used to route the log to the corresponding dataset.
+
+For example, if you want to route the {{edot-cf}} logs to custom datasets, you can add the following attributes to the log records:
+
+```yaml
+processors:
+  transform:
+    - set(attributes["data_stream.dataset"], "aws.cloudtrail") where attributes["aws.cloudtrail.event_id"] != nil
```

Comment:
At first glance it doesn't seem to fit the syntax: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor#general-config

error when running this processor:
```
remote stats will be ignored because of empty telemetry Elasticsearch URL
failed to get config: cannot unmarshal the configuration: decoding failed due to the following error(s):

'processors' decoding failed due to the following error(s):

'[transform/foo]' expected type 'map[string]interface {}', got unconvertible type '[]interface {}'
BenchmarkOTelbench/logs-otlp-1                 0                       NaN ns/op
failed to get config: cannot unmarshal the configuration: decoding failed due to the following error(s):
```